### ISSUE TITLE
Elementum fails to start again, possibly due to the lack of ipv6 support

### ISSUE THREAD

**Issue by pitsi** [elgatito/plugin.video.elementum#272]
This may be related to #141 so please, merge the 2 issues if you judge they are the same. 

So, I opened my kodi a few days ago and elementum's message came up saying "elementum daemon could not be started, retrying...". After one or two retries, elementum daemon was successfully started.
But since yesterday the elementum daemon fails to start at all. Here is the relevant log (available for 7 more days)
http://paste.debian.net/1051942/

Enabling ipv6 support makes the elementum daemon start as usual.

Elementum 0.0.61 on kodi 17.6 on debian testing x64.

p.s. While having ipv6 enabled, I noticed elementum was updated to 0.0.62 from 0.0.61...

**Comment by dankec2000** [comment]
Same here, Confirm that.

**Comment by dankec2000** [comment]
Here: http://ix.io/1s7W

**Comment by elgatito** [comment]
Still no errors. Like OS is killing the program without any errors.

**Comment by pitsi** [comment]
I will post a debug log tomorrow, I am off to bed in a while :(
Where is the "enable ipv6" option in elementum's settings?

**Comment by elgatito** [comment]
There is no, you are right, it is determined automatically now, and I see it has detected you have no IPv6, and sent that to the torrent library.

**Comment by elgatito** [comment]
@pitsi @dankec2000 Can you try to disable DHT option in `Elementum > Bittorrent` settings? Then if not helped - try to disable UTP in that tab.

**Comment by dankec2000** [comment]
Nope, still crashing. Tried with dht disabled, utp disabled and both disabled. Still the same

**Comment by pitsi** [comment]
Same thing as @dankec2000 said. Plus it makes no difference if I disable them both or one at a time.

**Comment by pep87** [comment]
Hi elgatito, please can you give me your email or telegram, i need to talk with you by private message for business

Thanks in advance 

My regards 

Joseph

**Comment by pitsi** [comment]
@pep87 
Why did you have to spam MY issue report with such irrelevant stuff? Github itself offers a form of email contact to every user, so rtfm!

**Comment by pep87** [comment]
Ok i deleted the message, excuse me

**Comment by dankec2000** [comment]
Is there any way to download v60 while we are waiting for possible v62 fix. I couldn't find v60 .zip anywhere...

**Comment by pitsi** [comment]
@dankec2000 
There you go
https://github.com/elgatito/plugin.video.elementum/releases/tag/v0.0.60
But why v60 and not v61?

@pep87 
Thank you. However, I can not "unsend" my report to github, sorry. I am leaving my message there until github sorts everything out, just as a proof that I did made a report and because the report contained a link to your message.

**Comment by pep87** [comment]
Ok no problem I understand you.

**Comment by dankec2000** [comment]
I meant v61, sorry. Yes, with auto update off and back to v61 everything's work great here.

**Comment by pitsi** [comment]
There you go, log with dht disabled :)
http://paste.debian.net/1052247/

**Comment by elgatito** [comment]
@pitsi log is not in debug mode and I don't see Elementum was trying to load, maybe disabled addon?

**Comment by pitsi** [comment]
There you go, kodi's log with debug enabled
https://pastebin.com/M2siVa6E

Sorry for the delay, but debian's pastebin (paste.debian.net) only accepts files smaller than 150k and that one was 160+k.
On the previous log, kodi was still running when I sent the log to pastebin. And I had enabled elementum's option to only log errors... because I thought it would help more :/ 
Is there an option to make an elementum-only log file?

**Comment by elgatito** [comment]
Issue in torrent library repository is here - https://github.com/anacrolix/torrent/issues/290

**Comment by elgatito** [comment]
Please, try 0.0.63 and post new log file.

**Comment by elgatito** [comment]
@pitsi can you disable DHT and post new log file, please?

**Comment by pitsi** [comment]
I do not know what the log says, but dht is disabled here

![screenshot000](https://user-images.githubusercontent.com/5880037/48982310-6fe31780-f0e9-11e8-8d76-59a63a8e48b0.png)

**Comment by elgatito** [comment]
@pitsi thanks. How do you disable/enable IPv6 on your system? I'm asking because I can't reproduce the error, so want to try something else.

**Comment by pitsi** [comment]
With this kernel parameter (shown in bold)

```
$ cat /proc/cmdline 
BOOT_IMAGE=/boot/vmlinuz-4.18.0-2-amd64 root=/dev/sda3 ro nomodeset noresume net.ifnames=0 **ipv6.disable=1** libahci.ignore_sss=1 quiet
```

**Comment by pitsi** [comment]
And the mandatory log, now that there is an upgrade
https://pastebin.com/j93wff6R

Kodi 18 rc1, elementum 0.0.64 and debug enabled.

If you need more info on where or how to use the forementioned kernel parameter, ask me.
